frontend/src/js/prime-web-receiver-pre.js

```diff
@@ -262,20 +279,74 @@ async function processOrgName(){
     return orgName;
 }
 
+function titleCase(str) {
+    str = str.toLowerCase().split(' ');
+    for (var i = 0; i < str.length; i++) {
+      str[i] = str[i].charAt(0).toUpperCase() + str[i].slice(1); 
+    }
+    return str.join(' ');
+  }
+
+async function processReportFeeds(){
+    const reportFeeds = document.getElementById("reportFeeds");  
+    var feeds = []; 
+    if( reportFeeds )
+        feeds = await fetchReportFeeds();
+        console.log( feeds );
+        console.log( `found ${feeds.length} feeds`)
+        if( feeds.length > 0 ){
+            feeds.forEach( (feed, idx ) => {
+                console.log( `adding feed ${feed}` )
+                
+                reportFeeds.innerHTML += `
+                    <input type="radio" id="tab${idx}" name="tabGroup1" class="tab" ${idx>0? "" : "checked"}>
+                    <label for="tab${idx}">${feed.replaceAll( "-", " ").toUpperCase()}</label>
+                    ` 
```

Comment:
To me this looks like it will create a tab for every department, even those with just one feed. Do I have that right? If so, I suggest that we should do the work to not create tabs when there is a single feed for a department.